### ISSUE TITLE
When scheduling event as non-user / public-> redirected to login page

### ISSUE THREAD

**Issue by bolwerk** [YunoHost-Apps/opensondage_ynh#82]
After a fresh install and upgrade - - YunoHost version: 4.3.6.2 - we have an error with the last step. We get redirected to the login page instead of creating the event. 

`https://transitional.anarchaserver.org/date/`

We also see in the logs / this problem.
[(https://github.com/YunoHost-Apps/opensondage_ynh/issues/50#issuecomment-546374143)]

But for us PHP solution didn't help but we also presume that is not the nature of the error.

**Comment by b01xy** [comment]
yes, same problem here : opensondage drives to the admin page of the poll when loggued as a yunohost user
but it drives to yunohost sso portal when not loggued as a user
the application has been set as accessible to the public
yunohost is installed inside a LXC container with an apache proxy on the host, could be problem of headers / redirection ?

**Comment by b01xy** [comment]
a workaround consist of desactivating the captive portal for this url in `/etc/nginx/conf.d/mydomain.org.conf`

````
#    access_by_lua_file /usr/share/ssowat/access.lua;

    include /etc/nginx/conf.d/mydomain.org.d/*.conf;

#    include /etc/nginx/conf.d/yunohost_sso.conf.inc;
#    include /etc/nginx/conf.d/yunohost_admin.conf.inc;
#    include /etc/nginx/conf.d/yunohost_api.conf.inc;
````

**Comment by b01xy** [comment]
A solution is to add in the admin panel > Users > Visitors : the permission ```OpenSondage (admin)``` to allow an anymous user / visitor to access this last page that sum-up the sondage 

So we could close this ticket as this is not a bug but a not super evident feature :=)
